AsyncDisplayKit/Layout/ASLayoutSpec.h

```diff
@@ -45,7 +45,7 @@ NS_ASSUME_NONNULL_BEGIN
  * setChild:forIdentifier: internally. For example, ASBackgroundLayoutSpec exposes a backgroundChild
```

Comment:
Does the comment need to mention anything that this does internally?  Seems odd.